enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -591,10 +592,29 @@ def can_redeem(self, lms_user_id, content_key, skip_customer_user_check=False):
 
         # learner not associated to enterprise
         if not skip_customer_user_check:
-            if self.lms_api_client.get_enterprise_user(self.enterprise_customer_uuid, lms_user_id) is None:
```

Comment:
I'm surprised this is None and not False 🤔 🤷

Reply:
This got changed with Katrina's earlier changes!